src/components/pages/Authentication/index.tsx

```diff
@@ -38,9 +39,11 @@ function Authentication({isLoading}) {
         </Authentication.Header>
         <Authentication.Content>
           {
-            !display
-              ? <Login />
-              : <SignUp isLoading={isLoading}/>
+            display == 2
```

Comment:
This is where I made 2 the ForgotPassword the default display. Would be changed once Login is merged